assets/style/style.css

```diff
@@ -2884,3 +2884,423 @@ select.form-control option {
 #foot {
     z-index: 200;
 }
+
+
+
+/* Flipdown styling (flipdown.js library) */
```

Comment:
This should be included as an npm  dependencyinstead

Reply:
I'm aware. We tried it. But it has no ES6 support so it does not work.